### DISCUSSION TITLE
dp.Page(label=...) is not working

### DISCUSSION THREAD

**Discussion by silviolima07** [datapane/datapane#105] [Help] [answered]
I´ve already built dp.Page using label in two reports but this week it is not working:
Could someone help me:
pagina1 = dp.Page(
       label="Dashes",
       blocks=[
               "#### Heatmap de Vagas pelo Brasil", 
               dp.Plot(mapa),
               "#### Total Vagas", 
               dp.Plot(fig3),
               "#### Total Vagas por Estado", 
               dp.Plot(fig1),
               "#### Total Vagas por Nível", 
               dp.Plot(fig2)
               ]
     )

label is not accepted.

**Comment by lanthias** [comment, marked as answer]
Thanks for the note and apologies for the bug. We renamed `label` to `title` on pages - as this was a breaking change, we have now pushed a fix whereby you will get a deprecation notice instead of it throwing an error; that said, I would recommend migrating `label` to `title` as we will remove it in the future.

**Comment by silviolima07** [comment]
Thanks

Em qua., 28 de abr. de 2021 às 17:09, Leo Anthias ***@***.***>
escreveu:
